scripts/cmake_configurator/parser/cmake_string.py

```diff
@@ -0,0 +1,41 @@
+import re
+
+class cmake_string(str):
```

Comment:
OK, let me explain here the idea, other than using `str` as data and not inheriting from it and calling the class `cmake_row` (or something like it): the row should be parsed in the construtor `__init__(self, row)`. So there shouldn't exist things like parse_variable(). You parse the row ONCE and then you can get its type (comment, empty, variable, other) and it's content (variable object, text in comment etc.). Get it?

Reply:
Done